conda/recipes/librmm/meta.yaml

```
@@ -17,11 +17,7 @@ requirements:
     - ninja
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    {% if cuda_major == "11" %}
-    - {{ compiler('cuda') }} {{ cuda_version }}
-    {% else %}
-    - {{ compiler('cuda') }}
-    {% endif %}
+    - {{ compiler('cuda') }} {{ cuda_version }} if cuda_major == "11" else compiler('cuda') }}
```

If we made `cuda-version` constrain `nvcc`, then we might be able to simplify like this:
```suggestion
    - {{ compiler('cuda') }}
```

@jakirkham Do you have thoughts on that? I know you've backported some changes to `nvcc` recently. I looked at the [recipe](https://github.com/conda-forge/nvcc-feedstock/blob/main/recipe/meta.yaml) and it doesn't seem to have `cuda-version` as a constraint right now.

---

This is how things work currently

* [`cuda-version` constrains `cudatoolkit`]( https://github.com/conda-forge/cuda-version-feedstock/blob/4722c6a1a7f01b9546d1118a1618aa2099d03cea/recipe/meta.yaml#L13-L18 )
* [`nvcc` depends on `cudatoolkit`]( https://github.com/conda-forge/nvcc-feedstock/blob/6de62063c45045de06e982ae3d6c536a4dc1c1ef/recipe/meta.yaml#L36 )
* [Old `nvcc`s also depend on `cudatoolkit` via a repodata patch]( https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/blob/834bbef5165b8bc8c5655abe933174f185b31442/recipe/patch_yaml/nvcc.yaml#L1-L29 )

So that should already give us the behavior we need

---

Oh good! I thought it had to be constrained somehow, but I couldn't see that from a quick glance. Thanks for checking into this. Let's try making that simplification.